backend/data/processors/conditions_preprocessing.py

```diff
@@ -13,6 +13,46 @@
 PREPROCESSED_CONDITIONS = {}
 CODE_MAPPING = read_data("data/utility/programCodeMappings.json")["title_to_code"]
 
+SPECIALISATION_MAPPINGS = {
+    'School of the Arts and Media honours': 'MDIA?H',
```

Comment:
Where are these strings being sourced from?

Reply:
we had to dig for them :/

Reply:
Yeah not ideal but not sure how else we'll do it :(